src/main/java/org/synyx/urlaubsverwaltung/person/web/PersonDetailsBasedataDtoMapper.java

```diff
@@ -0,0 +1,13 @@
+package org.synyx.urlaubsverwaltung.person.web;
+
+import org.synyx.urlaubsverwaltung.person.basedata.PersonBasedata;
```

Comment:
why is this in another package? Put it next to the controller :)

Reply:
This Mapper is used in `PersonDetailsViewController` which resides in the same package.